app/build.gradle.kts

```diff
@@ -30,9 +30,16 @@ val versionMajor = 0 // 0..Infinity
 val versionMinor = 1 // 0..9
 val versionPatch = 1 // 0..9
 
+java {
+    toolchain {
```

Comment:
I don't know this Toolchain. What is it for?

Reply:
https://docs.gradle.org/current/userguide/toolchains.html

If source, target and Kotlin's JVMTarget has been set to 11, we just make sure everything builds on 11 and there will be no java class version incompatibilities. Also helps with reproduceable builds if we ever decide go that way.